src/main/java/com/example/domainmodelopdrachtspringbootles11/controller/TelevisionController.java

```diff
@@ -0,0 +1,67 @@
+package com.example.domainmodelopdrachtspringbootles11.controller;
+
+import com.example.domainmodelopdrachtspringbootles11.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+    ArrayList<String >televisionDataBase = new ArrayList<>();
```

Comment:
Voor deze opdracht is het niet meer de bedoeling dat je een ArrayList gebruikt. We gebruiken hier nu de Repository, die we via constructor injection in de class injecteren.